api/models/users-model.js

```diff
@@ -10,7 +10,28 @@ function getUsersById(id) {
     .first();
 }
 
+function insert(user) {
+  return db('users')
+    .insert(user)
+    .then(ids => ({ id: ids[0] }));
```

Comment:
For this to work in PG, an .insert must have a second parameter which would be what you would like returned.
Seeing as you already have a getUsersById function, I would make this entire code something like this;

async function addUser(user) {
    const id = await db('users').insert(user, 'id')

    return getUsersById(id)
}

Reply:
Sorry, I forgot some brackets.

This is what it could be.
`async function addUser(user) {
  const [id] = await db('users').insert(user, 'id');

  return getUsersById(id);
}`